homeassistant/components/tahoma/cover.py

```diff
@@ -143,9 +173,7 @@ def is_closed(self):
     @property
     def device_class(self):
         """Return the class of the device."""
-        if self.tahoma_device.type == 'io:WindowOpenerVeluxIOComponent':
-            return 'window'
-        return None
+        return TAHOMA_DEVICE_CLASSES.get(self.tahoma_device.type, None)
```

Comment:
`get` will return `None` if not found.
```suggestion
        return TAHOMA_DEVICE_CLASSES.get(self.tahoma_device.type)
```